src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RandomNumberGenerator.cs

```diff
@@ -146,6 +146,29 @@ public static int GetInt32(int toExclusive)
             return GetInt32(0, toExclusive);
         }
 
+        /// <summary>
+        /// Creates an array of bytes with a cryptographically strong random sequence of values.
+        /// </summary>
+        /// <param name="byteCount">The number of bytes of random values to create.</param>
+        /// <returns>
+        /// An array populated with cryptographically strong random values.
+        /// </returns>
+        /// <exception cref="ArgumentOutOfRangeException">
+        /// The <paramref name="byteCount" /> parameter is less than zero.
+        /// </exception>
+        public static byte[] GetBytes(int byteCount)
+        {
+            if (byteCount < 0)
+                throw new ArgumentOutOfRangeException(nameof(byteCount), SR.ArgumentOutOfRange_NeedNonNegNum);
+
+            if (byteCount == 0)
+                return Array.Empty<byte>();
```

Comment:
Nit: consider swapping the order of these ifs.  If 0 is passed in, then we won't need to do the < 0 check.  If anything else is passed in, we need to do both checks anyway.